plugins/ros/src/components/riScInfo/riScStatus/RiScStatusComponent.tsx

```diff
@@ -108,73 +133,88 @@ export const RiScStatusComponent = ({
     setDifferenceFetchState(emptyDifferenceFetchState);
   }, [selectedRiSc]);
 
-  const [status, setStatus] = useState<0 | 1 | 2 | 3>(0);
+  const [status, setStatus] = useState<RiScStatusEnumType>(
+    RiScStatusEnum.CREATED,
+  );
 
   useEffect(() => {
     if (selectedRiSc.content.scenarios.length === 0) {
-      setStatus(0);
+      setStatus(RiScStatusEnum.CREATED);
     } else if (selectedRiSc.status === RiScStatus.Draft) {
-      setStatus(1);
+      setStatus(RiScStatusEnum.DRAFT);
     } else if (selectedRiSc.status === RiScStatus.SentForApproval) {
-      setStatus(2);
+      setStatus(RiScStatusEnum.WAITING);
     } else if (selectedRiSc.status === RiScStatus.Published) {
-      setStatus(3);
+      setStatus(RiScStatusEnum.PUBLISHED);
     }
   }, [selectedRiSc.status, selectedRiSc.content.scenarios]);
 
   const migration = selectedRiSc.migrationStatus?.migrationChanges;
 
+  const lastPublishedDateTime = selectedRiSc.lastPublished?.dateTime;
+  const lastPublishedNumberOfCommits =
+    selectedRiSc.lastPublished?.numberOfCommits;
+
+  const daysSinceLastModified = lastPublishedDateTime
+    ? calculateDaysSince(new Date(lastPublishedDateTime))
+    : null;
+
+  const numOfCommitsBehind =
+    typeof lastPublishedNumberOfCommits === 'number' &&
+    lastPublishedNumberOfCommits >= 0
+      ? lastPublishedNumberOfCommits
+      : null;
```

Comment:
Fortsatt litt usikker på hva som er poenget med disse linjene. Kan `lastPublishedNumberOfCommits` ta så mange forskjellige verdier? Sånn det ser ut for meg, så vil denne enten være ett tall, `null` eller `undefined` og den burde ikke kunne bli negativ. Så kanskje vi bare kan bruke `lastPublishedNumberOfCommits` direkte? Eventuelt, om vi ønsker at denne kun kan være et tall eller `null`, så kan vi gjøre følgende:

```suggestion
  const numOfCommitsBehind = lastPublishedNumberOfCommits ?? null
```